File: reimi/backend/reimi_app/src/main/java/com/reimi/reimi_app/domain/model/report/value/ForecastType.java
package com.reimi.reimi_app.domain.model.report.value;

public enum ForecastType {

    NO_CHANGE("今と変化なさそう"),
    IMPROVING("回復しそう"),
    DETERIORATING("下り坂になりそう");

    private final String label;

    ForecastType(String label) {
        this.label = label;
    }

    public String getLabel() {
        return label;
    }

}

File: reimi/frontend/reimi_app/lib/domain/value_objects/forecast_type.dart
import 'package:freezed_annotation/freezed_annotation.dart';

enum ForecastType {
  @JsonValue('NO_CHANGE')
  noChange, // 今と変化なさそう
  @JsonValue('IMPROVING')
  improving, // 回復しそう
  @JsonValue('DETERIORATING')
  deteriorating, // 下り坂になりそう
}
